docker-compose.yml

```
@@ -50,30 +59,51 @@ services:
       - riakdb
 
   zookeeper:
-    image: docker.io/confluentinc/cp-zookeeper:${CONFLUENT_PLATFORM_VERSION}
+    image: docker.io/confluentinc/cp-zookeeper:${ZOOKEEPER_VERSION}
+    healthcheck:
+      test: echo ruok | nc 127.0.0.1 2181 || exit -1
+      interval: 5s
+      timeout: 240s #🍎
+      retries: 50
     environment:
+      KAFKA_OPTS: "-Dzookeeper.4lw.commands.whitelist=ruok"
       ZOOKEEPER_CLIENT_PORT: 2181
 
   kafka1: &kafka-broker
-    image: docker.io/confluentinc/cp-kafka:${CONFLUENT_PLATFORM_VERSION}
+    image: wurstmeister/kafka:${KAFKA_VERSION}
```

Для тех, кто страдает podman'ом:
```suggestion
    image: docker.io/wurstmeister/kafka:${KAFKA_VERSION}
```